### ISSUE TITLE
Approval by klicking lamp not needed any more

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#644]
```
Hi,

since 0.9.19 players don't need to give approval by clicking the lamp any more. 
This only has disadvantages:

- Players can't control when to start the game, the whole group is extradicted 
to one player (host)
- Lamp should indicate whether a player is ready. If it's not clicked, there is 
a reason for it: a) player not ready, b) player afk, c) player does not know 
how to play. For those who don't know how to play: They find out in a few 
seconds that they have to click lamp. All the other players only have 
disadvantages. Only the host decides, when to start a game. Besides that, I 
think that during a gaming session, there should always be some seconds of 
break between the matches. I have no more control about this, and that is 
really bad.

I really hope that you guys fix it in upcoming versions. This change is 
annoying for anyone who played hedgewars more than one time. 

:)
```

Original issue reported on code.google.com by `another....@googlemail.com` on 1 Jun 2013 at 9:39

**Comment by GoogleCodeExporter** [comment]
```
I'm not a huge fan of this, but it does avoid my getting kicked when spectating 
or just having died early and on another screen when they were preparing for a 
new game.

Anyway, regardless, was done deliberately, and apparently based on people 
complaining about the exact opposite situation.
Impossible to please everyone.

Unless one group is more vocal than the other, going to have to flag this 
invalid.

changeset:   8384:e691acd778f5
user:        dag10
date:        Mon Jan 21 02:21:04 2013 -0500
summary:     It is now possible to start a game even if not all players are 
"ready"; a confirmation prompt is shown. This commit updates the server.


```

Original comment by `kyberneticist@gmail.com` on 1 Jun 2013 at 3:06
* Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
Hello and thanks for your report.
There are pro and cons on the change and it's difficult to please everyone.
We might tune out the delay between matches in future release to let everyone 
return to the lobby before starting a game.
```

Original comment by `vittorio...@gmail.com` on 1 Jun 2013 at 9:10
* Changed state: **WontFix**
